lambda_uploader/uploader.py

```diff
@@ -171,3 +174,16 @@ def _validate_package_size(self, pkg):
         '''
         if path.getsize(pkg) > MAX_PACKAGE_SIZE:
             LOG.warning("Size of your deployment package is larger than 50MB!")
+
+    def _format_vpc_config(self):
+        '''
+        Returns {} if the VPC config is set to None by Config,
+        returns the formatted config otherwise
+        '''
+        if self._config.raw['vpc']:
```

Comment:
That's what I thought as well, but what actually happened was that I was getting `__getattr__` errors on this thing not existing. Maybe I'm missing a detail, but using `.raw` solved this whole thing. When I was not using `.raw`, it looked like `set_defaults` was not executed at all.